charts/kargo/templates/api/deployment.yaml

```diff
@@ -97,20 +97,46 @@ spec:
 {{- end }}
             initialDelaySeconds: 5
 {{- end }}
-{{- if or .Values.kubeconfigSecrets.kargo (and .Values.api.oidc.enabled .Values.api.oidc.dex.enabled) .Values.api.tls.enabled }}
+{{- if or .Values.kubeconfigSecrets.kargo (and .Values.api.oidc.enabled .Values.api.oidc.dex.enabled) .Values.api.tls.enabled .Values.api.cabundle.enabled }}
           volumeMounts:
+          {{- if or .Values.kubeconfigSecrets.kargo (and .Values.api.oidc.enabled .Values.api.oidc.dex.enabled) .Values.api.tls.enabled }}
             - mountPath: /etc/kargo
               name: config
               readOnly: true
+          {{- end }}
+          {{- if .Values.api.cabundle.enabled }}
+            - mountPath: /etc/ssl/certs
+              name: certs
+          {{- end }}
 {{- end }}
 {{- with .Values.api.securityContext | default .Values.global.securityContext }}
           securityContext:
             {{- toYaml . | nindent 12 }}
 {{- end }}
           resources:
             {{- toYaml .Values.api.resources | nindent 12 }}
-{{- if or .Values.kubeconfigSecrets.kargo (and .Values.api.oidc.enabled .Values.api.oidc.dex.enabled) .Values.api.tls.enabled }}
+      {{- if .Values.api.cabundle.enabled }}
+      initContainers:
+      - name: parse-cabundle
+        image: docker.io/alpine/curl:latest
+        command:
+        - "/bin/sh"
+        - "-c"
+        args:
+        - |
+          for FILE in /tmp/source/*; do awk 'BEGIN {c=0;} /BEGIN CERT/{c++} { print > "/usr/local/share/ca-certificates/cert." c ".crt"}' < $FILE; done
```

Comment:
If this logic _doesn't_ move into the binary as proposed [here](https://github.com/akuity/kargo/pull/2413/files#r1720467608), then there is a bug in this script that will need fixing.

Try this with more than one cert in `/tmp/source` and see what happens. 😄

Reply:
I thought I was, ok I'll test it out ...

Reply:
Your counter resets every iteration, so each iteration can be overwriting files from the previous.

Reply:
I had to reinstall my environment, will be implementing this same technique in another project and so will be able to test soon.